packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -428,8 +445,11 @@ sdk.server('https://eu.api.example.com/v14');`),
 
     const config: TsConfigJson = {
       compilerOptions: {
-        module: 'NodeNext',
+        esModuleInterop: true,
+        module: 'ESNext',
+        moduleResolution: 'Bundler',
         resolveJsonModule: true,
+        strict: true,
```

Comment:
kind of surprised that the sdks we're generating are passing strict mode

Reply:
:cry-party: